### DISCUSSION TITLE
Enterprise OAuth providers

### DISCUSSION THREAD

**Discussion by mstade** [supabase/supabase#1174] [Questions] [answered]
Hi again!

On the pricing page there's a note in the PAYG tier column about enterprise OAuth providers, but I can't find any more information about this anywhere. Is there a blog post or any other information you can share about this feature? Specifically I'm interested in knowing if "Enterprise OAuth providers" equates to support for OpenID Connect and thus (hopefully) asymmetric keys?

**Comment by kiwicopple** [comment, marked as answer]
We're still building out a lot of the Providers. OAuth providers will be things like Azure, SAML, SSO, and yes, OpenID Connect.

If you are looking for specific functionality I recommend opening an issue on github.com/supabase/gotrue so that we can schedule the work on it.

**Reply by mstade** [reply]
Will definitely do so, thanks! If I can find the time I'd be happy to contribute a first stab at an OIDC provider as well, if that's something y'all will entertain?

**Reply by kiwicopple** [reply]
We'd definitely entertain that 🙏

**Reply by Riz-waan** [reply]
Is there an issue on gotrue I can upvote? I want to know if I would be able to use my custom auth provider with supabase, or something like SSO.

**Comment by rafa-niuco** [comment]
Hey! Any updates?
